src/state/migrations/1.test.ts

```diff
@@ -43,7 +43,7 @@ describe('migration to v1', () => {
     expect(result?.user?.tokens).toEqual({})
     expect(result?.user?.pairs).toEqual({})
     expect(result?.user?.timestamp).toEqual(previousState.user?.timestamp)
-    expect(result?.user?.hideBaseWalletBanner).toEqual(false)
+    expect(result?.user?.hideAndroidAnnouncementBanner).toEqual(false)
```

Comment:
I don't think the assertion on every other unrelated state key is necessary--should prob just delete the whole line instead of swapping out the `hideBaseWalletBanner` state for android